libs/shared/src/utils/geo-utils.ts

```diff
@@ -303,3 +303,4591 @@ export const states = [
+    { name: 'Omsk', code: 'Omsk' },
+    { name: 'Orenburg', code: 'Orenburg' },
+    { name: 'Orjol', code: 'Orjol' },
+    { name: 'Penza', code: 'Penza' },
+    { name: 'Perm', code: 'Perm' },
+    { name: 'Primorje', code: 'Primorje' },
+    { name: 'Pskov', code: 'Pskov' },
+    { name: 'Pskovskaya Oblast', code: 'Pskovskaya Oblast' },
+    { name: 'Rjazan', code: 'Rjazan' },
+    { name: 'Rostov', code: 'Rostov' },
+    { name: 'Saha', code: 'Saha' },
+    { name: 'Sahalin', code: 'Sahalin' },
+    { name: 'Samara', code: 'Samara' },
+    { name: 'Samarskaya', code: 'Samarskaya' },
+    { name: 'Sankt-Peterburg', code: 'Sankt-Peterburg' },
+    { name: 'Saratov', code: 'Saratov' },
+    { name: 'Smolensk', code: 'Smolensk' },
+    { name: 'Stavropol', code: 'Stavropol' },
+    { name: 'Sverdlovsk', code: 'Sverdlovsk' },
+    { name: 'Tajmyrija', code: 'Tajmyrija' },
+    { name: 'Tambov', code: 'Tambov' },
+    { name: 'Tatarstan', code: 'Tatarstan' },
+    { name: 'Tjumen', code: 'Tjumen' },
+    { name: 'Tomsk', code: 'Tomsk' },
+    { name: 'Tula', code: 'Tula' },
+    { name: 'Tver', code: 'Tver' },
+    { name: 'Tyva', code: 'Tyva' },
+    { name: 'Udmurtija', code: 'Udmurtija' },
+    { name: 'Uljanovsk', code: 'Uljanovsk' },
+    { name: 'Ulyanovskaya Oblast', code: 'Ulyanovskaya Oblast' },
+    { name: 'Ust-Orda', code: 'Ust-Orda' },
+    { name: 'Vladimir', code: 'Vladimir' },
+    { name: 'Volgograd', code: 'Volgograd' },
+    { name: 'Vologda', code: 'Vologda' },
+    { name: 'Voronezh', code: 'Voronezh' },
+]
+
+export const RW = [
+    { name: 'Butare', code: 'Butare' },
+    { name: 'Byumba', code: 'Byumba' },
+    { name: 'Cyangugu', code: 'Cyangugu' },
+    { name: 'Gikongoro', code: 'Gikongoro' },
+    { name: 'Gisenyi', code: 'Gisenyi' },
+    { name: 'Gitarama', code: 'Gitarama' },
+    { name: 'Kibungo', code: 'Kibungo' },
+    { name: 'Kibuye', code: 'Kibuye' },
+    { name: 'Kigali-ngali', code: 'Kigali-ngali' },
+    { name: 'Ruhengeri', code: 'Ruhengeri' },
+]
+
+export const ST = [{ name: 'Sao Tome and Principe', code: 'ST' }]
+
+export const SA = [
+    { name: 'Al Khobar', code: 'Al Khobar' },
+    { name: 'Aseer', code: 'Aseer' },
+    { name: 'Ash Sharqiyah', code: 'Ash Sharqiyah' },
+    { name: 'Asir', code: 'Asir' },
+    { name: 'Central Province', code: 'Central Province' },
+    { name: 'Eastern Province', code: 'Eastern Province' },
+    { name: "Ha''il", code: "Ha''il" },
+    { name: 'Jawf', code: 'Jawf' },
+    { name: 'Jizan', code: 'Jizan' },
+    { name: 'Makkah', code: 'Makkah' },
+    { name: 'Najran', code: 'Najran' },
+    { name: 'Qasim', code: 'Qasim' },
+    { name: 'Tabuk', code: 'Tabuk' },
+    { name: 'Western Province', code: 'Western Province' },
+    { name: 'al-Bahah', code: 'al-Bahah' },
+    { name: 'al-Hudud-ash-Shamaliyah', code: 'al-Hudud-ash-Shamaliyah' },
+    { name: 'al-Madinah', code: 'al-Madinah' },
+    { name: 'ar-Riyad', code: 'ar-Riyad' },
+]
```

Comment:
What's the source of this data? Because it's not accurate. See:
- [Ministry of Interior website (list on the left)](https://www.moi.gov.sa/wps/portal/Home/emirates/!ut/p/z1/04_Sj9CPykssy0xPLMnMz0vMAfIjo8ziLQPdnT08TIy83Q0dzQwcPc2N_A08TQ3dPY30wwkpiAJKG-AAjgZA_VFgJc7ujh4m5j4GBhY-7qYGno4eoUGWgcbGBo7GUAV4zCjIjTDIdFRUBAApuVo7/dz/d5/L2dJQSEvUUt3QS80TmxFL1o2XzBJNDRIMTQyS0dBREIwQUEzUTlLNUkxMEw0/)
- [Wikipedia](https://en.wikipedia.org/wiki/Provinces_of_Saudi_Arabia)